podman/tests/integration/test_container_create.py

```diff
@@ -322,6 +322,12 @@ def test_read_write_tmpfs(self):
                     print(inspect)
                     self.assertIn(expected_output, logs)
 
+    def test_container_working_dir_deprecation(self):
+        with self.assertWarns(PendingDeprecationWarning):
+            self.client.containers.create(
+                self.alpine_image, working_dir="/tmp", command=["/bin/ls", "-l", "/"]
+            )
+
```

Comment:
Thank you for including a test, but it will need to be adjusted to verify `workdir` works as expected.